Rick-and-Morty/Rick And Morty/Views/CharacterCell.swift

```diff
@@ -0,0 +1,49 @@
+//
+//  CharacterCell.swift
+//  Rick And Morty
+//
+//  Created by Scottie Gray on 2021-07-21.
+//  Copyright © 2021 Novoda. All rights reserved.
+//
+
+import SwiftUI
+
+struct CharacterCell: View {
+    let character: Character
+    
+    var body: some View {
+        HStack(spacing: 8) {
+            // left handside images for morties
+            if character is Morty {
+                CharacterCellImage(character: character)
+            }
+            
+            VStack(alignment: .leading, spacing: 8) {
+                Text(character.name)
+                Text(character.description)
+            }
+            // right handside images for ricks
+            if character is Rick {
+                CharacterCellImage(character: character)
+            }
```

Comment:
I think we should extract this logic from view. all the view needs to know is if the image should be right or left. It does not need to know if its rick or Morty. What if we. would add additional character that image would be on the left as well? We would need to add logic here.